apiary.apib

```diff
@@ -0,0 +1,347 @@
+FORMAT: 1A
+HOST: http://deploy.cwp.govt.nz
+
+# Deploynaut API
+
+This API is a newly developed interface for the CWP Dashboard. Data structure is based on the
+[JSON API specifications](http://jsonapi.org/).
+
+
+# Using the API [/naut]
+
+In order to use the API, you will need to acquire a token via Service Desk request. In order to authenticate,
```

Comment:
Just say "Use HTTP Basic Authentication" - this is the most crucial piece of information. Basic auth is supported by [all](http://docs.guzzlephp.org/en/latest/request-options.html#auth) [clients](https://curl.haxx.se/docs/manpage.html#--basic) I [can](https://linux.die.net/man/1/ab) [think](https://docs.silverstripe.org/en/4/developer_guides/integration/restfulservice/#basic-authenication) of. I've never in my career had to build the `Authorization` header by hand, so let's leave the example and explanation to other sources, which can do it in full.